tests/py/test_api_multi_gpu.py

```diff
@@ -0,0 +1,97 @@
+import unittest
+import trtorch
+import torch
+import torchvision.models as models
+
+from model_test_case import ModelTestCase
+
+gpu_id = 1
```

Comment:
theres no point in making this redundant declaration we can just set self.gpu_id to 1.

Reply:
Single point of configuration. Using same variable at all the places in the test script